Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -109,11 +109,11 @@ def test_bottle
     formula_file = CoreFormulaRepository.new.formula_dir/"testball.rb"
     formula_file.write <<-EOS.undent
       class Testball < Formula
-        url "https://example.com/testabll-0.1.tar.gz"
+        url "https://example.com/testball-0.1.tar.gz"
       end
     EOS
     HOMEBREW_CACHE.cd do
-      assert_match /testball-0\.1.*\.bottle\.tar\.gz/,
+      assert_match %r{testball-0.1.*.bottle.tar.gz},
```

Comment:
The backslash in `\.` is important. It cannot be omitted. Without it, dot will match any character.